code/modules/reagents/reagent_containers/_glass.dm

```diff
@@ -19,7 +19,8 @@
 	pickup_sound = 'sound/foley/bottlepickup1.ogg'
 	watertight = FALSE // /glass uses the open container flag for this
 
-	var/list/can_be_placed_into = list(
+/obj/item/chems/glass/proc/get_atoms_can_be_placed_into()
```

Comment:
not super sure why this was necessary? it complicates overriding it and runs into the potential of overrides accidentally causing list mutations affecting all procs if they do `. = ..(); . -= /obj/machinery/whatever` to remove something from a subtype. obviously an edge case but one that's annoying enough i'm a little worried